README.md

```diff
@@ -80,11 +80,11 @@ We recommend running Arlo on Ubuntu 20.
 
 - Clone the Arlo repository from https://github.com/votingworks/arlo.
 - `make dev-environment` or, if you prefer, look at individual make tasks like `deps`, `initdevdb`, `install-development`, and `resetdb`
+- Ensure `poetry` is available in your `PATH`. This will vary slightly depending on your shell and where `poetry` was installed. With bash, that should look like: `export PATH="$PATH:$HOME/.local/bin"`, which you can add to your `.bashrc` or `.bash_profile`.
```

Comment:
Probably worth dropping `bash_profile` as `bashrc` will be the right move. My error!